src/env-config.js

```diff
@@ -22,6 +22,11 @@ const envConfig = {
       && ['false', 'disable'].includes(process.env.VUE_APP_WEBTRIT_NAVIGATION_DRAWER_MINIMIZING.toLowerCase())
   ),
 
+  webtritRecordingDisablePlayButton: !(
```

Comment:
After renaming env variable to `VUE_APP_WEBTRIT_RECORDING_PLAY_BUTTON_DISABLE` this method will be `webtritRecordingPlayButtonDisable`.